main.go

```diff
@@ -28,26 +32,32 @@ const (
 type Configs struct {
 	ProjectLocation string `env:"project_location,dir"`
 	APKPathPattern  string `env:"apk_path_pattern"`
+	AppPathPattern  string `env:"app_path_pattern"`
 	Variant         string `env:"variant"`
 	Module          string `env:"module"`
+	BuildType       string `env:"build_type,opt[apk,aab]"`
 	Arguments       string `env:"arguments"`
 	CacheLevel      string `env:"cache_level,opt[none,only_deps,all]"`
 	DeployDir       string `env:"BITRISE_DEPLOY_DIR,dir"`
 }
 
-func getArtifacts(gradleProject gradle.Project, started time.Time, pattern string, includeModule bool) (artifacts []gradle.Artifact, err error) {
-	artifacts, err = gradleProject.FindArtifacts(started, pattern, includeModule)
-	if err != nil {
-		return
+func getArtifacts(gradleProject gradle.Project, started time.Time, patterns []string, includeModule bool) (artifacts []gradle.Artifact, err error) {
+	for _, pattern := range patterns {
+		afs, err := gradleProject.FindArtifacts(started, pattern, includeModule)
+		if err != nil {
+			continue
```

Comment:
pls print a warning, to make the issue debugable

Reply:
Fixed

Reply:
You could also add `pattern` to the log, possibly between delimiters, so as to make everything evident.

e.g.: `Failed to find artifact (path/with/strange/
whitespace/error), error: %s"`

Reply:
Fixed in:
https://github.com/bitrise-steplib/bitrise-step-android-build/pull/11/commits/0412b8b5af341311e798570d67f4163b28ae5ecc